### PR TITLE
fix: storybook for shimmer + spinner stories now in a single folder

### PR DESCRIPTION
**Pull request by LinKCoding**
### Overview
See [WEB-1868](https://www.google.com/url?q=https://codecademy.atlassian.net/browse/WEB-1868&sa=D&source=docs&ust=1677617907717646&usg=AOvVaw1B_Ipmk5nnff6eRnIA37IJ).

This PR adds:
1. Creates a folder, `Loaders` in `styleguide/Atoms/Loaders`  to store `Spinner.stories.mdx` and `Shimmer.stories.mdx`  (<- previously incorrectly placed Brand Labs)
2. For Shimmer, enabled sandbox to work and linked a WIP Figma for Shimmer. 

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [WEB-1868](https://www.google.com/url?q=https://codecademy.atlassian.net/browse/WEB-1868&sa=D&source=docs&ust=1677617907717646&usg=AOvVaw1B_Ipmk5nnff6eRnIA37IJ) 
- [ ] I have run this code to verify it works

#### Testing Instructions
Visit:
1. Spinner Story: https://640624f9d5a0af10455f86b9--gamut-preview.netlify.app/?path=/docs/atoms-loaders-spinner--spinner
2. Shimmer Story: https://640624f9d5a0af10455f86b9--gamut-preview.netlify.app/?path=/docs/atoms-loaders-shimmer--shimmer 

#### PR Links and Envs
This is a storybook change, doesn't affect any front-end code. 

[WEB-1868]: https://codecademy.atlassian.net/browse/WEB-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WEB-1868]: https://codecademy.atlassian.net/browse/WEB-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.290-alpha.ccd01a.0<br/>@codecademy/gamut-labs@35.6.5-alpha.ccd01a.0<br/>@codecademy/gamut@52.0.9-alpha.ccd01a.0<br/>@codecademy/monaco@0.7.9-alpha.ccd01a.0<br/>@codecademy/styleguide@63.5.15-alpha.ccd01a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://640624f9d5a0af10455f86b9--gamut-preview.netlify.app>https://640624f9d5a0af10455f86b9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/640624f9d5a0af10455f86b9>Deploy Logs</a></p>

File: packages/gamut/src/ConnectedForm/SubmitButton.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { FormState } from 'react-hook-form';
 
 import { Box, FlexBox } from '../Box';
 import { ButtonProps, FillButton } from '../Button';
-import { Spinner } from '../Spinner';
+import { Spinner } from '../Loading/Spinner';
 import { useSubmitState } from './utils';
```
Comment (dreamwasp) [inline packages/gamut/src/ConnectedForm/SubmitButton.tsx:8]: ah didn't realize this component was being moved! i would still add the alpha package to the monolith or portal and push up a test pr just to double-check everything is compiling correctly. 

i'm not expecting this to be a breaking change - just good practice

**Review by dreamwasp — COMMENTED** [review]
stories look great, just wanna double check the alpha package in portal or the monolith

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.291-alpha.2b997a.0<br/>@codecademy/gamut-labs@35.7.1-alpha.2b997a.0<br/>@codecademy/gamut@52.1.1-alpha.2b997a.0<br/>@codecademy/monaco@0.7.10-alpha.2b997a.0<br/>@codecademy/styleguide@63.6.1-alpha.2b997a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6408a4ff29a7fb5a78407b8e--gamut-preview.netlify.app>https://6408a4ff29a7fb5a78407b8e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6408a4ff29a7fb5a78407b8e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.291-alpha.ea6e82.0<br/>@codecademy/gamut-labs@35.7.1-alpha.ea6e82.0<br/>@codecademy/gamut@52.1.1-alpha.ea6e82.0<br/>@codecademy/monaco@0.7.10-alpha.ea6e82.0<br/>@codecademy/styleguide@63.6.1-alpha.ea6e82.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://640b796087ceca1a74428552--gamut-preview.netlify.app>https://640b796087ceca1a74428552--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/640b796087ceca1a74428552>Deploy Logs</a></p>

File: packages/gamut/src/ConnectedForm/SubmitButton.tsx (hunk 1 above)
Comment (LinKCoding) [reply packages/gamut/src/ConnectedForm/SubmitButton.tsx:8]: https://tengu.codecademy.com/user-form?PR_ENV=portal-app-pr-1457
it works!

**Pull request merged**